lib/lambda_ethereum_consensus/telemetry.ex

```diff
@@ -118,7 +119,10 @@ defmodule LambdaEthereumConsensus.Telemetry do
       last_value("vm.system_counts.atom_count"),
       last_value("vm.system_counts.port_count"),
       last_value("vm.message_queue.length", tags: [:process]),
-      last_value("vm.uptime.total", unit: :millisecond)
+      last_value("vm.uptime.total", unit: :millisecond),
+
+      # Db Metrics
+      last_value("db.size.total")
```

Comment:
We can add the unit:

```suggestion
      last_value("db.size.total", unit: :byte)
```